WMF Framework/EventPlatformClientLibrary.swift

```diff
@@ -190,28 +190,10 @@ public class EPC: NSObject {
      */
     private let legacyEventLoggingService: EventLoggingService
 
-    /**
-     * Store events until the library is finished initializing
-     *
-     * The EPC library makes an HTTP request to a remote stream configuration
-     * service for information about how to evaluate incoming event data. Until
-     * this initialization is complete, we store any incoming events in this
-     * buffer.
-     *
-     * Only modify (append events to, remove events from) *synchronously* via
-     * `queue.sync`
-     */
-    private var inputBuffer: [EventRequest] = []
-
-    /**
-     * Maximum number of events allowed in the input buffer
-     */
-    private let inbutBufferLimit = 128
```

Comment:
Just wanted to point out that we'll now have no in-memory event limit since the input buffer was the only one that seemed to have a max. Probably moot though since your persistence is coming, we'll just need to be sure there's logic in that to prevent events from piling up indefinitely.

Reply:
Hmm, yeah, probably best to leave this limit in place in the meantime.  I'll put it back in.